### ISSUE TITLE
Compiling apk for Android fails

### ISSUE THREAD

**Issue by farver** [freshplanet/ANE-Facebook#259]
Hi freshplanet

Thanks for the useful ANE! I seem to have an issue compiling for Android with the latest binary, though.
I can compile a working iOS app with AIR SDK compiler v 29.0.0.112. Trying to compile for Android, however, I get these errors:

```
aapt tool failed:/private/var/folders/1d/3t28g89j2w1cjb0_kjz7pcrw0000gn/T/298d28f6-11fa-472d-a1b1-9b8c52f03737/AirFacebook-res/values/values.xml:161: error: Error retrieving parent for item: No resource found that matches the given name '@style/Theme.AppCompat.NoActionBar'.

/private/var/folders/1d/3t28g89j2w1cjb0_kjz7pcrw0000gn/T/298d28f6-11fa-472d-a1b1-9b8c52f03737/AirFacebook-res/values/values.xml:166: error: Error retrieving parent for item: No resource found that matches the given name '@style/Theme.AppCompat.Dialog'.
```

Am I doing something wrong? Or is there something missing in the binary?

**Comment by Kelsonk** [comment]
I also have this problem. Really could use a fix.

**Comment by mumeka** [comment]
Yes same problem related to AppCompat. A fix can be cool

**Comment by franmocad** [comment]
Same problem. Any solution?

**Comment by leandrop20** [comment]
Same problem :(

**Comment by mtilhan** [comment]
same problem

**Comment by CaptainHug** [comment]
I've made the changes needed in a fork here and included an updated bin: https://github.com/CaptainHug/ANE-Facebook

You will need to replace some things in the AIRSDK folder too, as it has been built with an older version of the linker. You can find what you need here: https://github.com/CaptainHug/ANE-Facebook/blob/master/ios/AIRSDK_overlay.zip

**Comment by mumeka** [comment]
![error](https://user-images.githubusercontent.com/3039771/43453401-b4ded4d8-94c2-11e8-8eb3-436ce0fdb939.png)

There is still error at android packaging I don't why this is happening.

**Comment by CaptainHug** [comment]
Looks like you are including another ANE that already includes some or all of the Android resources. You will most likely need a different build, commenting out the Android resources that you already have included in your project from the build/platform-android.xml file, but retaining the iOS changes.

**Comment by mumeka** [comment]
Yes after trying your compiled ane at a clean project it works, however it is conflicting with my current anes. I will try to delete some resources as you suggested.

**Comment by albermotion** [comment]
same problem here

**Comment by orcuncun** [comment]
How to resolve file conflicts within another ANE FİLES?

**Comment by rainbowcreatures** [comment]
Just to clarify these for anyone else, to git rid of those build errors you need to include AppCompatV7 and CardViewV7 ane's. Those can be found as part of other ANE packages(like distriqt), usually they are free.
